builders/buildPost.ts

```diff
@@ -14,14 +19,23 @@ const buildContent = (
   $: CheerioSelector
 ): { message: string; images: string[] } => {
   const userContent = $(".userContent");
-  let message = $("p", userContent).html() || "";
 
-  const textExposedRootPargaraph = $(".text_exposed_root > p");
-  if (textExposedRootPargaraph.length) {
-    const textExposedShow = $("div.text_exposed_show").html() || "";
-    message = `<p>${textExposedRootPargaraph.html()}</p>${textExposedShow}`;
-  }
+  // get all post paragraphs post text
+  const paragraphs = $("p", userContent).toArray();
 
+  // remove html leaving only <br>
+  let message = paragraphs.reduce(
```

Comment:
Do we still need `.replace(/<[^>]+>/g, "")` above then?

Reply:
There can be <br/> so better to leave this as it is at this point